pkg/pkgrepository/package_repo_app.go

```diff
@@ -96,7 +96,7 @@ metadata:
 	}
 
 	if pkgRepository.Spec.SyncPeriod == nil {
-		desiredApp.Spec.SyncPeriod = &metav1.Duration{Duration: time.Minute * 5}
+		desiredApp.Spec.SyncPeriod = &metav1.Duration{Duration: time.Minute * 10}
```

Comment:
Are we doing this for repositories as well? Was under impressions only PackageInstalls.